test/v1/connectors/ListBenefitsConnectorSpec.scala

```diff
@@ -86,6 +86,19 @@ class ListBenefitsConnectorSpec extends ConnectorSpec {
         .returns(Future.successful(response))
     }
 
+    def stubTysHttp(response: DownstreamOutcome[ListBenefitsResponse[HMRCStateBenefit, CustomerStateBenefit]],
+                    queryParams: Seq[(String, String)]): Unit = {
+      MockedHttpClient
```

Comment:
This can be updated to use the `willGet` method provided in the `ConnectorSpec`. I think you may need to update the method to take queryr params as an input (and give it a default value of an empty sequence)